app/build.gradle

```diff
@@ -186,6 +186,7 @@ dependencies {
     implementation "androidx.work:work-runtime:${workVersion}"
     implementation "androidx.work:work-rxjava2:${workVersion}"
     implementation 'androidx.swiperefreshlayout:swiperefreshlayout:1.1.0'
+    implementation 'androidx.exifinterface:exifinterface:1.3.0-rc01'
```

Comment:
@mahibi Any reason for using ```1.3.0-rc01```? I'd advice to use latest which according to https://developer.android.com/jetpack/androidx/releases/exifinterface is ```1.3.2```

Reply:
ah thx. came in via auto suggestion